desktop/hardened-desktop/enhanced-container-isolation/index.md

```diff
@@ -0,0 +1,94 @@
+
+### What happens when Enhanced Container Isolation is enabled?
+
+When Enhanced Container Isolation is enabled using [Admin Controls](../admin-controls/index.md), the following features are enabled: 
+
+- All user containers are automatically run in Linux User Namespaces which ensures stronger isolation.
+- The root user in the container maps to an unprivileged user at VM level.
+- Users can continue using containers as usual, including bind-mounting host directories, volumes, networking configurations, etc.
+- Privileged containers work, but they are only privileged within the container's Linux User Namespace, not in the Docker Desktop VM.
+- Containers can no longer share namespaces with the Docker Desktop VM. For example, `--network=host`, `--pid=host`.
+- Containers can no longer modify configuration files in the Docker Desktop VM
+- Console access to the Desktop VM is forbidden for all users
+- Containers become harder to breach. For example, sensitive system calls are vetted and portions of `/proc` and `/sys` are emulated.
+
+For more information on how Enhanced Container Isolation work, see [How does it work?](how-eci-works.md).
+
+>Important
+>
+>Enhanced Container Isolation is currently incompatible with WSL and does not protect Kubernetes pods. For more information on known limitations and workarounds, see [FAQS and known issues](faq.md).
+{: .important}
+
+### How do I enable Enhanced Container Isolation?
+
+As an admin, you first need to [configure a `registry.json` file to enforce sign-in](../../../docker-hub/configure-sign-in.md). This is because this feature requires a Docker Business subscription and therefore your Docker Desktop users must authenticate to your organization for this configuration to take effect.
+
+Next, you must [create and configure the `admin-settings.json` file](../admin-controls/configure-ac.md) and specify:
+
+```JSON
+{
+ "enhancedContainerIsolation": {
+    "value": true,
+    "locked": true
+    }
+}
+```
+
+Once this is done, developers need to either quit, re-launch, and sign in to Docker Desktop, or launch and sign in to Docker Desktop for the first time.
+
+>Important
+  >
+  >Selecting **Restart** from the Whale menu isn't enough as it only restarts some components of Docker Desktop.
+  {: .important}
+
+### What do users see when this setting is enforced?
+
+When Enhanced Container Isolation is enabled, users see that containers run within a Linux user-namespace. For example:
+
+```
+$ docker run -it --rm alpine
+/ # cat /proc/self/uid_map 
+         0     100000      65536
+```
+
+This indicates that the container's root user (0) maps to unprivileged user (100000) in the Docker Desktop VM, and that the mapping extends for a range of 64K user-IDs.
+
+In contrast, without Enhanced Container Isolation the Linux user-namespace is not used:
+
+```
+$ docker run -it --rm alpine             
+/ # cat /proc/self/uid_map                           
+         0          0 4294967295
+```
```

Comment:
Make this command a one-liner?

```suggestion
```console
$ docker run -it --rm alpine cat /proc/self/uid_map                           
         0          0 4294967295
```
```